checker/src/abstract_value.rs

```diff
@@ -847,6 +847,15 @@ impl Path {
                         // can refine this value.
                     }
                 }
+            } else {
```

Comment:
nit: I have to scroll back 20 pages to see what this is an else for, a comment about it would have saved me 2.5s of my life.

Reply:
Comment duly added.